students/jvirtue/lesson03/mailroom.py

```diff
@@ -0,0 +1,86 @@
+#Lesson 3 Assignment 4
+#Mailroom Part Assignment
+#Jason Virtue 01/26/2019
+#UW Self Paced Python Course
+
+
+#Thank You Letter Section
+def Send_Thanks(db_name):
+    response = "Not a donor or the word list"
+    while response not in db_name:
+        print("----------------------------------------")
+        response = input("What is the full name of the donor?>(Name, List, Exit) ")
+        response = response.title()
+        if response == "Exit":
+            break
+        elif response == "List":
+            print("----------------------------------------")
+            print("Here are our Donors:> ")
+            print(donor_list(db_name))
+        elif response != "List":
+            print("----------------------------------------")
+            amount = int(input("How much money did {} donate?> ".format(response)))
+            if response not in donor_list(db_name):
+                donors_db = donor_add(response,db_name)
+                donors_db = amount_add(response, amount, db_name)
+            else:
+                donors_db = amount_add(response, amount, db_name)
+            print("----------------------------------------")
+            print(thank_you(response, amount))
+            print(db_name)
+
+def donor_list(db_name):
+    return [item[0] for item in db_name]
+
+def donor_add(name,db_name):
+    return db_name.append([name])
+
+def amount_add(name, amt, db_name):
+    return db_name[donor_list(db_name).index(name)].append(amt)
+
+def thank_you(person,amount):
+    return "Thank you {} ".format(person) + "for the donation in the amount of {}!".format(amount)
+
+#Create a report
+def Sum_Gift(donor, db_name):
+    total = 0
+    for i in db_name:
+        if i[0] == donor:
+            total = sum(i[1:])
+    return total
+
+def donation_count(donor, db_name):
+    count = 0
+    for i in db_name:
+        if i[0] == donor:
+            count = len(i[1:])
```

Comment:
you want to either break or return to avoid continue looping once you found the right donor